src/ser.rs

```diff
@@ -124,10 +124,6 @@ pub enum Error {
     #[doc(hidden)]
     KeyNewline,
 
-    /// Arrays in TOML must have a homogenous type, but a heterogeneous array
-    /// was emitted.
-    ArrayMixedType,
```

Comment:
Oh my mistake, this is the public variant which would be great to stick around for now